Exiled.API/Features/Roles/IHumeShieldRole.cs

```diff
@@ -0,0 +1,15 @@
+namespace Exiled.API.Features.Roles
+{
+    using PlayerRoles.PlayableScps.HumeShield;
+
+    /// <summary>
+    /// Represents a role that supports a hume shield.
+    /// </summary>
+    public interface IHumeShieldRole
+    {
+        /// <summary>
+        /// Gets a reference to the role's <see cref="HumeShieldModuleBase"/>.
+        /// </summary>
+        public HumeShieldModuleBase HumeShieldBase { get; }
```

Comment:
```suggestion
        HumeShieldModuleBase HumeShieldBase { get; }
```
Interface access modifiers are public by default.